packages/contracts-bedrock/deploy/021-SystemDictatorSteps-2.ts

```diff
@@ -221,17 +166,59 @@ const deployFn: DeployFunction = async (hre) => {
     )
   }
 
-  // Step 5 initializes all contracts.
-  await doStep({
+  await doPhase({
     isLiveDeployer,
     SystemDictator,
-    step: 5,
+    phase: 2,
     message: `
+      Phase 2 consists of the following steps:
+
+      Step 3 will clear out some legacy state from the AddressManager. Once you execute this step,
+      you WILL NOT BE ABLE TO RESTART THE SYSTEM using exit1(). You should confirm that the L2
+      system is entirely operational before executing this step.
```

Comment:
The problem here is that the L2OO isn't initialized, so "make sure the L2 system is entirely operational" isn't quite possible. https://github.com/ethereum-optimism/optimism/pull/5253 is an attempt to fix this, but it needs changes to work correctly

Reply:
You're right. This also came up on that PR. 

I'm not sure we actually want to change the step numbers, but it does feel to me like the `updateDynamicConfig()` call is best treated as if it was `phase2` in 3 (rather than 2) step process.